compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/lower_to_ukernel.mlir

```diff
@@ -1,5 +1,3 @@
-// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(func.func(iree-amdaie-lower-to-ukernels,cse,canonicalize))" %s | FileCheck %s
-// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(func.func(iree-amdaie-lower-to-ukernels{pass-pipeline=simple-pack path-to-ukernels="/custom/path/to/ukernels"},cse,canonicalize))" %s | FileCheck %s --check-prefix=SIMPLE_PACK_AND_UK_PATH
 // RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(func.func(iree-amdaie-lower-to-ukernels{pass-pipeline=pad-pack},cse,canonicalize))" %s | FileCheck %s --check-prefix=PAD_PACK
```

Comment:
Since others are removed, please add a `path-to-ukernels="/custom/path/to/ukernels"` like what was there previously.
And replace all uses of `mm.o` with `/custom/path/to/ukernels/mm.o`.
It should just work.

Reply:
Done.